Sources/PT.PM.Common/CommonUtils.cs

```diff
@@ -43,39 +43,29 @@ public static bool TryConvertToInt64(this string value, int fromBase, out long r
             false;
 #endif
 
-        public static bool IsRunningOnLinux
+        public static bool IsWindows
         {
             get
             {
-                if (isRunningOnLinux == null)
+                if (!isRunningOnLinux.HasValue)
                 {
-                    int p = (int)Environment.OSVersion.Platform;
-                    isRunningOnLinux = (p == 4) || (p == 6) || (p == 128);
+                    isRunningOnLinux = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
i think it's bad idea to assign result of `IsOsPlatform(Windows)`, to `isRunningOnLinux`. Also we can have `OSX`. Better to call `IsOsPlatform(OsPlatform.Linux)`

Reply:
Yes, it's a misprint.